### PR TITLE
chore: CHANGELOG for 0.13.0

### PR DESCRIPTION
**Pull request by angelozerr**
chore: CHANGELOG for 0.13.0

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,19 @@
 # Tools for MicroProfile Changelog
 
+## [0.13.0](https://github.com/redhat-developer/vscode-microprofile/milestone/14?closed=1) (October 17, 2024)
+
+### Bug Fixes
+
+- Cannot publish to OpenVSX due to out of date NodeJS. See [#303](https://github.com/redhat-developer/vscode-microprofile/issues/303).
```
Comment (rgrunber) [inline CHANGELOG.md:7]: "Update to NodeJS 20"

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,19 @@
...
+- REST endpoint codelens is not resolving a `public static final` parameter. See [lsp4mp#467](https://github.com/eclipse/lsp4mp/pull/467).
```
Comment (rgrunber) [inline CHANGELOG.md:8]: Just make sure to copy any of the changes from the lsp4mp change to these entries.

File: CHANGELOG.md (hunk 1 above)
Comment (angelozerr) [reply CHANGELOG.md:7]: added

File: CHANGELOG.md (hunk 2 above)
Comment (angelozerr) [reply CHANGELOG.md]: fixed

**Review by datho7561 — APPROVED** [review]

**Pull request merged**